_data/sidebars/docs_sidebar.yml

```diff
@@ -85,6 +85,10 @@ entries:
             url: /installation-source-installation.html
             output: web, pdf
 
+          - title: Discoverability
```

Comment:
I don't really like the name, but I see the difficulty of choosing a good one... :sweat_smile: No better suggestion at the moment.

Reply:
What about "Finding preCICE"? Style-wise this would fit to "Buidling from source" and "Linking to preCICE".
And it would reflect the CMake terminology `find_package` etc.

Reply:
How about just "Finding", then?
We already mix e.g., "Instal**lation**" and "Build**ing**"
"Discoverability" is anyway a property (getting tricky to explain what I mean).

Reply:
Both are not great as they can easily be confused with a guide on finding the dependencies.

Why is naming so difficult...

Reply:
I would never think that any of the names proposed so far, and at that place in the structure, would mean finding the dependencies, especially since they have their own prominent page above. I think it makes sense if we look at the order:

```
Preparation
Dependencies
Configuration
Building
Testing
Installation
Finding
Advanced
```

I would maybe even bring consistency to all (only on the sidebar, to not break any links):

```
Preparation
Dependencies
Configuring
Building
Testing
Installing
Finding
Advanced
```

Reply:
Looking at all the arguments raised so far, I would go with `Finding` (fits with `Building` and `Testing`, is one word, fits with the CMake `find_package`, pkg-config also uses the term "to find a library").

In a follow-up, we could change the sidebar entries for consistency (but not the permalinks, please).

I will change the name and merge.